servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -305,7 +310,8 @@ public CreatePathResult createPath(TachyonURI path, CreatePathOptions options)
     currentInodeDirectory.setLastModificationTimeMs(options.getOperationTimeMs());
 
     LOG.debug("createFile: File Created: {} parent: ", lastInode, currentInodeDirectory);
-    return new CreatePathResult(modifiedInodes, createdInodes, traversalResult.getPersisted());
+    return new CreatePathResult(Lists.newArrayList(traversalResult.getInode()), createdInodes,
```

Comment:
`modifiedInodes` serves the same function as `traversalInode.getInode()`.

Have we returned anything other than `CreatePathResult` originally?